test/fork.js

```diff
@@ -33,6 +43,7 @@ test('resolves promise with tests info', function (t) {
 		});
 });
 
+/*
```

Comment:
Oh,
I forgot.
I think this test can be dropped.
There is no longer a way to test a non-existent test file passed to `fork`, because `fork` now requires the file be precompiled. There is already a test for a non-existing files in `api.js`